packages/office-addin-debugging/src/start.ts

```diff
@@ -8,10 +8,13 @@ import * as devSettings from "office-addin-dev-settings";
 import { DebuggingMethod } from "office-addin-dev-settings";
 import * as manifest from "office-addin-manifest";
 import * as nodeDebugger from "office-addin-node-debugger";
+import * as os from "os";
+import * as path from "path";
 import { getProcessIdsForPort } from "./port";
 import { startDetachedProcess, startProcess  } from "./process";
 
-export const processIdFile = "devServerProcessId.txt";
+const processIdFile = "devServerProcessId.txt";
+export const processIdFilePath = path.join(os.tmpdir(), processIdFile);
```

Comment:
It would be better to export as a function, e.g. getDebuggingInfoPath(). It would also be good to add a JSDoc comment which would provide intellisense info.

Reply:
It might help to move functions related to reading, writing, deleting the debugging info file into its own file, and providing the functions related to it there. That can then be used by both start.ts and stop.ts